requirements/scenarios.md

```diff
@@ -26,26 +26,26 @@ However, they do wish to add attestations to the validity of the content.
       - The `SBoM` and `src` artifacts are created as reference types to the image, creating a graph of artifacts.
-      - Each of the artifacts are signed with the Notary v2 wabbit-networks key.
-1. The Wabbit Networks signed contents are pushed to a public OCI compliant registry.
+      - Each of the artifacts are signed with the wabbit-networks key.
+2. The Wabbit Networks signed contents are pushed to a public OCI compliant registry.
       - Docker can provide an additional Docker hub signature providing an extra level of certification confidence.
-1. ACME Rockets consumes the `net-monitor` software, importing the referenced artifacts into their private registry.
+3. ACME Rockets consumes the `net-monitor` software, importing the referenced artifacts into their private registry.
       - ACME Rockets verifies the content, including additional scanning and functional testing for their environment.
       - The SBoM is trusted as they trust artifacts signed by wabbit-networks, or possibly defer trust to the Docker Hub certification signature.
       - They denote verification of the SBoM and scanning with an ACME Rockets signature.
       - A `deploy` artifact, referencing a specific configuration definition, may also be signed and saved, providing a historical record of what was deployed.
-1. The ACME Rockets environment may enforce various policies prior to deployment:
+4. The ACME Rockets environment may enforce various policies prior to deployment:
       - Evaluating the content in the `SBoM` for policies on specific packages.
       - ACME Rockets only allows content signed by ACME Rockets to be deployed, and only from the registry identified in the ACME Rockets signature.
       - Once validated, the `src` and `SBoM` are no longer needed for deployment allowing the `image` to be deployed separately with it's own signature.
-1. Once the policy manager completes its validation (k8s ingress controller with OPA), the deployment to the hosting environment is initiated.
+5. Once the policy manager completes its validation (k8s ingress controller with OPA), the deployment to the hosting environment is initiated.
       - ACME Rockets runs in an air-gapped environment, requiring all key access to be resolved within their environment.
 
 **Implications of this requirement:**
 
 - Signatures can be placed on any type of [artifact](artifacts-repo) stored in an OCI compliant registry using an [OCI Manifest][oci-manifest]
 - Signatures can be persisted as references to the [OCI Manifest][oci-manifest], allowing a entity to define a collection of artifacts.
-- Signatures and their public keys can be moved within, and across OCI compliant registries which support Notary v2.
+- Signatures and their public keys can be moved within, and across OCI compliant registries which support Notary signature.
```

Comment:
> which support Notary signature

Is that supposed to be "Notary signatures" or "a Notary signature". It's not clear if it's one, many, or some other meaning.

Reply:
I will update it to `the Notary signature` which is defined by the Notary signature specification. cc @toddysm  for comments.

Reply:
As clarified https://github.com/notaryproject/.github/issues/35, it should be `Notary Project signature`